host_configuration_scripts/3dsmax/3dsmax-2024.ps1

```diff
@@ -0,0 +1,19 @@
+mkdir C:\3dsmax_setup
+
+Write-Host " --- Downloading from S3 --- "
+aws s3 cp --no-progress s3://<your-bucket-name>/resources/3ds_max_2024_full.zip C:\3dsmax_setup\3dsmax.zip
```

Comment:
```suggestion
aws s3 cp --no-progress $env:3DS_MAX_ZIP_FILE_S3_URI C:\3dsmax_setup\3dsmax.zip
```
Can we make the entire S3 URI pull from an environment variable instead? This gives the user flexibility for where the zip file is